features/app/config/routes.yaml

```diff
@@ -4,6 +4,10 @@ secured_document:
     defaults:
         _signed: true
 
-sign:
-    path: /sign
-    controller: CoopTilleuls\UrlSignerBundle\Tests\Controller\SignController
+sign-path:
+    path: /sign-path
+    controller: CoopTilleuls\UrlSignerBundle\Tests\Controller\SignPathController
```

Comment:
Wouldn't it be better to have only one controller and a query parameter to control the generated signed URL?

Reply:
I thought about it at first and it felt a little bit "hacky" to me, but I can change it if you want. You tell me!